### PR TITLE
add option to disable locale prefix routes

### LINKED ISSUE #476: Option to disable prefixed routes
Option to disable routing in routes.php (https://github.com/rainlab/translate-plugin/blob/master/routes.php#L9) to prevent duplicate URLs for SEO if is used custom plugin to set language or language is set depending on browser.

### PR DESCRIPTION
**Pull request by mjauvin**
New config to disable locale prefix routes (disableLocalePrefixRoutes). Defaults to false.

Resolves #476

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -26,4 +25,14 @@
     */
     'cacheTimeout'        => 1440,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Disable locale prefix routes
+    |--------------------------------------------------------------------------
+    |
+    | Should we disable the locale prefix routes?
```
Comment (LukeTowers) [inline config/config.php]: Please change to 

> Disables the automatically generated locale prefixed routes (i.e. /en/original-route) when enabled.

File: config/config.php [hunk 2]
```diff
@@ -8,7 +8,6 @@
     |
     | Should the default locale be prefixed by the plugin?
     |
-    | ( For now only used in the alternate-hreflang component )
     |
```
Comment (LukeTowers) [inline config/config.php]: Please remove another line

File: routes.php [hunk 3]
```diff
@@ -22,6 +22,10 @@
         return;
     }
 
+    if (Config::get('rainlab.translate::disableLocalePrefixRoutes', false)) {
```
Comment (Samuell1) [inline routes.php]: I think this can be moved on top of this function to prevent useless calls for App and Translator.

**Comment by mjauvin** [comment]
@Samuell1 can you test this please?

**Comment by Samuell1** [comment]
@mjauvin Tested and it works 👍

**Pull request merged**